glib/PtrArray.cs

```diff
@@ -91,6 +91,15 @@ public void Dispose ()
 		[DllImport ("libglib-2.0-0.dll", CallingConvention = CallingConvention.Cdecl)]
 		static extern void g_free (IntPtr item);
 
+		static ListElementFree defaultFree;
+		protected static ListElementFree DefaultFree {
```

Comment:
👍 

There are enough free funcs in the world that i don't want to have to guess which one i'm invoking ;p